### PR TITLE
fix alignment issue in mc ping.

### PR DESCRIPTION
**Pull request by sinhaashish**
The mc ping out was mis-aligned .
In order to have a proper alignment , i ave fixed the size of each colums as below
the  size of min , max , average is fixed of 8 length, while counter and error are of 3 lengths.

the o/p looks like 
```
ashish@ashish:~/code/go/src/github/minio/mc$ ./mc ping play
  1: https://play.min.io:443   min=3.45s      max=3.45s      average=3.45s      errors=0     roundtrip=3.45s
  2: https://play.min.io:443   min=363.34ms   max=3.45s      average=1.90s      errors=0     roundtrip=363.34ms
  3: https://play.min.io:443   min=301.88ms   max=3.45s      average=1.37s      errors=0     roundtrip=301.88ms
  4: https://play.min.io:443   min=301.88ms   max=3.45s      average=1.11s      errors=0     roundtrip=322.43ms
  5: https://play.min.io:443   min=277.85ms   max=3.45s      average=942.13ms   errors=0     roundtrip=277.85ms
  6: https://play.min.io:443   min=277.85ms   max=3.45s      average=861.73ms   errors=0     roundtrip=459.71ms
  7: https://play.min.io:443   min=275.79ms   max=3.45s      average=778.03ms   errors=0     roundtrip=275.79ms
  8: https://play.min.io:443   min=275.79ms   max=3.45s      average=725.44ms   errors=0     roundtrip=357.32ms
  9: https://play.min.io:443   min=275.79ms   max=3.45s      average=676.34ms   errors=0     roundtrip=283.59ms
 10: https://play.min.io:443   min=275.79ms   max=3.45s      average=636.37ms   errors=0     roundtrip=276.60ms
 11: https://play.min.io:443   min=275.79ms   max=3.45s      average=657.86ms   errors=0     roundtrip=872.77ms
 12: https://play.min.io:443   min=275.79ms   max=3.45s      average=663.11ms   errors=0     roundtrip=720.81ms
```

@harshavardhana  @donatello  : please suggest any better approach, if any

### CODE REVIEW

**Comment by donatello** [comment]
It might be easier to do this alignment dynamically using lipgloss using JoinHorizontal

**Comment by sinhaashish** [comment]
> It might be easier to do this alignment dynamically using lipgloss using JoinHorizontal

JoinHorizontal didn't worked. The present approach will work for second and milliseconds. It pads white space . IMO this is a good approach . It solves the alignment problem.

**Comment by sinhaashish** [comment]
@donatello @kannappanr  PTAL

**Comment by kannappanr** [comment]
```
  1: https://play.min.io:443   min=898.97ms   max=898.97ms   average=898.97ms   errors=0   roundtrip=898.97ms
  2: https://play.min.io:443   min=220.85ms   max=898.97ms   average=559.91ms   errors=0   roundtrip=220.85ms
```
Alignment looks fine. Why is the port displayed here?

**Comment by kannappanr** [comment]
Debug option is crashing the binary

File: cmd/ping.go [hunk 1]
```diff
@@ -243,14 +243,37 @@ func trimToTwoDecimal(d time.Duration) string {
...
+		unit = pad("s", " ", 7-len(fmt.Sprintf("%.02f", f)), false)
 	default:
 		f = float64(d) / float64(time.Millisecond)
-		unit = "ms"
+		unit = pad("ms", " ", 6-len(fmt.Sprintf("%.02f", f)), false)
 	}
 	return fmt.Sprintf("%.02f%s", f, unit)
 }
 
+func pad(s, p string, count int, left bool) string {
```
Comment (donatello) [inline cmd/ping.go:254]: Some comments about what this is doing would be good - I can't understand what is being done here.

**Review by donatello — CHANGES_REQUESTED** [review]
Please fix conflicts

**Comment by sinhaashish** [comment]
> Debug option is crashing the binary

The debug is crashing because the `madmin-go` alive function is replying with a -ve response time in debug mode .
```
{Endpoint:https://play.min.io/ ResponseTime:-3.882013ms DNSResolveTime:4.088693ms Online:true Error:<nil>}
```

This might be beacuse of time being set in `ClientTrace` . Need more investigate on this

File: cmd/ping.go (hunk 1 above)
Comment (sinhaashish) [reply cmd/ping.go:261]: done

**Review by harshavardhana — APPROVED** [review]

**Comment by vadmeste** [comment]
@sinhaashish can you also add DNS output ?

**Comment by sinhaashish** [comment]
> @sinhaashish can you also add DNS output ?

DNS output is present in json mode. Since the DNS output has the value only once and  0 all other time , It didn't made any sense to display it in normal mode. Moreover the present op was discussed with AB , and DNS was not a part of it.

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**